build/DependencyVersions.props

```diff
@@ -44,5 +51,6 @@
   <PropertyGroup>
     <VersionToolsVersion>1.0.27-prerelease-01723-01</VersionToolsVersion>
     <DotnetDebToolVersion>2.0.0-preview2-25331-01</DotnetDebToolVersion>
+    <BuildTasksFeedToolVersion>1.0.0-prerelease-02203-01</BuildTasksFeedToolVersion>
```

Comment:
Shouldn't the BuildTasksFeedToolVersion be: 2.1.0-prerelease-02221-02
that is what is in release/15.5 now - or is the timing of the PR off a bit.